DESIGN.md

```diff
@@ -258,10 +259,10 @@ The process for writing a checker goes as follows:
  - Decide on check "type name" (as seen in the yaml)
  - Write a check struct that implements the [Checker](https://github.com/chasinglogic/redalert/blob/master/checks/checks.go#L20) interface
  - Implement [Argable](https://github.com/chasinglogic/redalert/blob/master/checks/checks.go#L15) for your struct 
-   - Often you can just copy the FromArgs of the example check above.
```

Comment:
This is correct, it is referencing the ExampleCheck

Reply:
Got it, I'll push something else that I think makes it clearer

Reply:
sgtm